packages/chart/src/Gauge.ts

```diff
@@ -249,6 +254,8 @@ export class Gauge extends SVGWidget {
 
     update(domNode: HTMLElement, element: d3SelectionType) {
         super.update(domNode, element);
+        element.style("cursor", this.click !== Gauge.prototype.click ? "pointer" : null);
```

Comment:
This is an interesting idea... maybe this same concept could be applied to other widgets with optional click events. Something like: 
```javascript
showPointer() {
    return this.click && this.click !== super.click; // I'm not sure if this exact condition would work...
}
```

Reply:
Yea - was pleased it worked - I suspect we can worry about generalizing when we find a second place that needs it.